trigger_systemtests.py

```diff
@@ -83,78 +83,81 @@ def determine_image_tag():
     else:
         return branch
 
-def generate_travis_job(adapter, user, trigger_failure = True):
 
-    triggered_by = os.environ["TRAVIS_JOB_WEB_URL"] if "TRAVIS_JOB_WEB_URL" in\
-         os.environ else "manual script call"
+def generate_travis_job(adapter, user, trigger_failure=True, branch="master"):
+    triggered_by = os.environ["TRAVIS_JOB_WEB_URL"] if "TRAVIS_JOB_WEB_URL" in \
+                                                       os.environ else "manual script call"
 
     base = adapters_info[adapter].base
 
     after_failure_action = "python push.py -t {TEST} --base {BASE} ;"
     main_test_script = "python system_testing.py -s {TEST} --base {BASE}"
 
-    base_remote = "precice/precice-{base}-develop".format(base = base.lower())
+    base_remote = "precice/precice-{base}-develop".format(base=base.lower())
     main_build_script = "docker build -f adapters/Dockerfile.{adapter} -t \
-        {user}/{adapter}:{tag} --build-arg from={base_remote} .".format(adapter =
-                adapters_info[adapter].repo, user = user, base_remote =
-                base_remote, tag = determine_image_tag())
+        {user}/{adapter}:{tag} --build-arg from={base_remote} .".format(adapter=
+                                                                                                adapters_info[
+                                                                                                    adapter].repo,
+                                                                        user=user, base_remote=
+                                                                                                base_remote,
+                                                                        tag=determine_image_tag())
 
     if trigger_failure:
         after_failure_action += " python trigger_systemtests.py --failure --owner {USER} --adapter {ADAPTER}"
```

Comment:
```suggestion
         after_failure_action += " python trigger_systemtests.py --failure --owner {USER} --adapter {ADAPTER}" + "--branch {}".format(os.environ["TRAVIS_BRANCH"])
```
See explanation above.